app/locales/en.json

```diff
@@ -16,6 +16,7 @@
     "forHour": "for an hour",
     "for2Hours": "for 2 hours",
     "for5Hours": "for 5 hours",
+    "untilMorning": "until morning",
```

Comment:
we will need to copy this to all localization files

Reply:
I thought that english translation will be used by default if there is no translation in the selected language (no entry in the file)?